### PR TITLE
Fix cqlsh history problem (bash hopefully as well)

### PR DESCRIPTION
**Pull request by a1exsh**
Given how cqlsh tries to find the history, the only way is to set HOME
to something where the default docker user has permissions to write.

This ends up on the data volume, but should not be an issue since if you
got one, you get a lot of the actual data already.

### CODE REVIEW

File: Dockerfiles/Cassandra-2/Dockerfile [hunk 1]
```diff
@@ -44,6 +44,9 @@ RUN rm -f /etc/cassandra/cassandra.yaml && chmod 0777 /etc/cassandra
 
 COPY planb-cassandra.sh /usr/local/bin/
 
+ENV HOME=/var/lib/cassandra
+WORKDIR /var/lib/cassandra
```
Comment (slitsche) [inline Dockerfiles/Cassandra-2/Dockerfile]: Why not `WORKDIR $HOME`?

Comment (a1exsh) [inline Dockerfiles/Cassandra-2/Dockerfile]: Didn't expect it to work, but it look that it does.

**Comment by slitsche** [comment]
👍

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**